Makefile

```diff
@@ -195,7 +195,9 @@ $(VM_IMAGE): $(TARFILE) $(NODE_CACHE) packaging/arch/PKGBUILD bots test/vm.insta
 		--script $(CURDIR)/test/vm.install $(TEST_OS)
 
 # convenience target for the above
-vm: $(VM_IMAGE)
+.PHONY: vm
```

Comment:
This is already in the `.PHONY` list at the bottom, so redundant.

Reply:
Thanks!